### ISSUE TITLE
Require specified prefix

### ISSUE THREAD

**Issue by ndmitchell** [ndmitchell/hlint#508]
A lot of companies require all their code to start:

```haskell
-- (C) Team Awesome, 2018, all rights reserved
-- Don't steal this code
```

There should be an `.hlint.yaml` setting to require a given prefix.

**Comment by rmehri01** [comment]
Hi, I'd like to give this a try if possible! I'm taking a look through `Yaml.hs` right now but I'm not quite sure where to start 😅

**Comment by ndmitchell** [comment]
Hmm - I think I might remove the good first issue label - since I'm not massively sure where to start either!

You likely want to follow something along the lines of https://github.com/ndmitchell/hlint/blob/master/src/Hint/Restrict.hs which uses information from the `.yaml` file to restrict stuff.  You probably do need to add a field to the Yaml.hs data type somewhere, but no idea quite where. And then examine it in a hint somewhere.

I'm also starting to wonder whether it's a good idea or not. Most projects that would have wanted this feature (e.g. commercial projects) end up being multi-language at some point, and then you need more than HLint anyway.

**Comment by rmehri01** [comment]
Hmm okay that makes sense. Do you think it is still worth trying to add then or no?

**Comment by ndmitchell** [comment]
Unless it's something you personally want to make use of, I probably wouldn't bother.
